<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowSystem</class>
 <widget class="QDialog" name="WindowSystem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1091</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#WindowSystem
{
	background-color:rgb(178,178,178,255);
}</string>
  </property>
  <widget class="QFrame" name="bottom">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1070</width>
     <height>701</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#bottom
{
	background-color:#ffffff;
 
	border:none;
	border-radius:25px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="WeatherStatus">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>350</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#282A49</string>
    </property>
    <property name="text">
     <string>晴朗</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="BackMain">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>20</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">max-width:50px;
 max-height:50px;
 min-width:50px;
 min-height:50px;
border-radius:25px;
background-color: white;
 border-style: solid;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../other/main.qrc">
      <normaloff>:/new/Main/Air/Back.png</normaloff>:/new/Main/Air/Back.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="CurrentTemp">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>321</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>200</pointsize>
     </font>
    </property>
    <property name="text">
     <string>24</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LocalArea">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#282A49</string>
    </property>
    <property name="text">
     <string>珠海市</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="MinTemp">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>420</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#282A49</string>
    </property>
    <property name="text">
     <string>16°</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="MaxTemp_">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>420</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#282A49</string>
    </property>
    <property name="text">
     <string>24°</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="XProgressBar" name="TempRange_Xprogress" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>390</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="ButtonBox">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>70</y>
      <width>551</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox#ButtonBox{
border:none;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="WindowClose">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>30</y>
       <width>131</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border-radius:15px;
	background:#FFFFFF;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/new/Curtain/Curtain/Down_OFF.png</normaloff>
       <normalon>:/new/Curtain/Curtain/Down_ON.png</normalon>:/new/Curtain/Curtain/Down_OFF.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>100</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="WindowStop">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>131</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border-radius:15px;
	background:#FFFFFF;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../other/main.qrc">
       <normaloff>:/new/Curtain/Curtain/Stop_OFF.png</normaloff>
       <normalon>:/new/Curtain/Curtain/Stop_ON.png</normalon>:/new/Curtain/Curtain/Stop_OFF.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>80</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="WindowOpen">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>30</y>
       <width>131</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border-radius:15px;
	background:#FFFFFF;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../other/main.qrc">
       <normaloff>:/new/Curtain/Curtain/Up_OFF.png</normaloff>
       <normalon>:/new/Curtain/Curtain/Up_ON.png</normalon>:/new/Curtain/Curtain/Up_OFF.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>120</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="XProgressBar" name="OutSideCo2_Xprogress" native="true">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>520</y>
      <width>421</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="XProgressBar" name="OutsideHumi_XprogressBar" native="true">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>610</y>
      <width>421</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="XProgressBar" name="OutSidePM25_Xprogress" native="true">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>420</y>
      <width>421</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="PM25P">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>390</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="Co2P">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>490</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="HumiP">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>590</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="MaxTemp_2">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>460</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PibotoLt</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#282A49</string>
    </property>
    <property name="text">
     <string>PM2.5</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="MaxTemp_3">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>560</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PibotoLt</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#282A49</string>
    </property>
    <property name="text">
     <string>CO2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="MaxTemp_4">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>650</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PibotoLt</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#282A49</string>
    </property>
    <property name="text">
     <string>湿度</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="TempHumi">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>490</y>
      <width>231</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>60</pointsize>
     </font>
    </property>
    <property name="text">
     <string>24/30</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="WeatherStatus_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>620</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#282A49</string>
    </property>
    <property name="text">
     <string>室内温湿度</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>470</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>570</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QSlider" name="Device_Qslider">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>330</y>
      <width>501</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>6</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="WeatherStatus_4">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>280</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#282A49</string>
    </property>
    <property name="text">
     <string>窗户：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="CurrentWindow">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>280</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#282A49</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="WeatherStatus_6">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>370</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#282A49</string>
    </property>
    <property name="text">
     <string>户外空气质量：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="WeatherStatus_7">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>370</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#282A49</string>
    </property>
    <property name="text">
     <string>优</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XProgressBar</class>
   <extends>QWidget</extends>
   <header>xprogressbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../other/main.qrc"/>
 </resources>
 <connections/>
</ui>
